index.ios.js

```diff
@@ -15,10 +15,24 @@ import { createStore, applyMiddleware } from 'redux';
 import rootReducer from './app/reducers/rootReducer.js';
 import ViewContainer from './app/components/ViewContainer.js';
 import Menu from './app/components/Menu.js';
-
+import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
+import promise from 'redux-promise';
 const SideMenu = require('react-native-side-menu');
 
-var store = createStore(rootReducer);
+//applies thunk middleware for async actions
```

Comment:
Can we remove these commented lines? And could we add a `TODO` for factoring out the `createStore` to it's own file?